go/test/endtoend/tabletmanager/throttler_topo/throttler_test.go

```diff
@@ -470,24 +509,24 @@ func TestCustomQuery(t *testing.T) {
 			}
 		})
 	})
-}
-
-func TestRestoreDefaultQuery(t *testing.T) {
-	// validte going back from custom-query to default-query (replication lag) still works
-	defer cluster.PanicHandler(t)
-
-	t.Run("enabling throttler with standard threshold", func(t *testing.T) {
-		_, err := onlineddl.UpdateThrottlerTopoConfig(clusterInstance, true, false, throttlerThreshold.Seconds(), "", false)
+	// validate going back from custom-query to default-query (replication lag) still works
+	t.Run("enabling throttler with default query and threshold", func(t *testing.T) {
+		_, err := throttler.UpdateThrottlerTopoConfig(clusterInstance, true, false, throttler.DefaultThreshold.Seconds(), throttler.DefaultQuery, false)
 		assert.NoError(t, err)
+
+		// Wait for the throttler to be up and running everywhere.
+		for _, tablet := range clusterInstance.Keyspaces[0].Shards[0].Vttablets {
+			throttler.WaitForThrottlerStatusEnabled(t, tablet, true, throttler.DefaultConfig, throttlerEnabledTimeout)
+		}
 	})
-	t.Run("validating OK response from throttler with low threshold, heartbeats running", func(t *testing.T) {
-		waitForThrottleCheckStatus(t, primaryTablet, http.StatusOK)
-	})
-	t.Run("validating pushback response from throttler on low threshold once heartbeats go stale", func(t *testing.T) {
-		time.Sleep(2 * onDemandHeartbeatDuration) // just... really wait long enough, make sure on-demand stops
```

Comment:
This line above is a critical part of the test: we want to see that after waiting `2 * onDemandHeartbeatDuration`, there is no more heartbeats. The sleep needs to be longer than `onDemandHeartbeatDuration`. By removing this line, the test becomes incorrect -- it does not test what it's supposed to be testing.